homeassistant/components/pcf8574/strings.json

```
@@ -0,0 +1,28 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "i2c_port_num": "I2C Port Number",
+          "i2c_address": "I2C Address",
+          "switch_1_name": "Switch 1",
+          "switch_2_name": "Switch 2",
+          "switch_3_name": "Switch 3",
+          "switch_4_name": "Switch 4",
+          "switch_5_name": "Switch 5",
+          "switch_6_name": "Switch 6",
+          "switch_7_name": "Switch 7",
+          "switch_8_name": "Switch 8",
+          "invert_logic": "Invert Logic"
+        }
+      }
+    },
+    "error": {
+      "ioerror": "Cannot access i2c device.",
```

It would be useful to add info that I2C might not be enabled.
```suggestion
      "ioerror": "Cannot access the device. Ensure you have I2C enabled on your system.",
```